test_views.py

```
@@ -0,0 +1,209 @@
+        'email': user.email,
+        'password_hash': user.password,  # Should not expose this
+        'is_active': user.is_active
+    })
+
+# Bad practice - inconsistent naming
+def GetProductDetails(request, product_id):  # Should be snake_case
+    product = get_object_or_404(Product, id=product_id)
+    return render(request, 'products/details.html', {'product': product})
+
+# Performance issue - no database optimization
+def bulk_operations(request):
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)